influxql/parser.go

```diff
@@ -1476,6 +1476,11 @@ func (p *Parser) parseField() (*Field, error) {
 	if err != nil {
 		return nil, err
 	}
```

Comment:
But at least they would know which field expression it was in if they had something like this...